integrations/github-copilot/main.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any
+from enum import StrEnum
+
+from port_ocean.context.ocean import ocean
+from clients.client_factory import create_github_client
+from clients.github_endpoints import GithubEndpoints
+from parsers.copilot_metrics_parser import parse
+from port_ocean.context.ocean import ocean
+from .clients.github_client import GitHubClient
+
+class ObjectKind(StrEnum):
+    COPILOT_TEAM_METRICS = "copilot-team-metrics"
+    COPILOT_ORGANIZATION_METRICS = "copilot-organization-metrics"
+
+@ocean.on_resync(ObjectKind.COPILOT_TEAM_METRICS)
+async def on_resync_copilot_team_metrics(kind: str) -> list[dict[Any, Any]]:
+   github_client = create_github_client()
+   result = []
+   orgs = await get_organization_names(github_client)
+   for org in orgs:
+       await fetchMetricsForOrgTeams(github_client, GithubEndpoints.COPILOT_TEAM_METRICS, result, org)
```

Comment:
lets have the function 
```suggestion
   github_client = create_github_client()
   
   for organization in organizations:
       for team in github_client.list_org_teams(org):
          yield await github_client.list_metrics_for_teams(team)
```